huaweicloud/services/acceptance/cfw/resource_huaweicloud_cfw_ip_address_group_member_test.go

```diff
@@ -0,0 +1,127 @@
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func getIPAddressGroupMemberResourceFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	region := acceptance.HW_REGION_NAME
+	// getIPAddressGroupMember: Query the CFW IP address group member detail
+	var (
+		getIPAddressGroupMemberHttpUrl = "v1/{project_id}/address-items"
+		getIPAddressGroupMemberProduct = "cfw"
+	)
+	getIPAddressGroupMemberClient, err := cfg.NewServiceClient(getIPAddressGroupMemberProduct, region)
+	if err != nil {
+		return nil, fmt.Errorf("error creating CFW Client: %s", err)
+	}
+
+	getIPAddressGroupMemberPath := getIPAddressGroupMemberClient.Endpoint + getIPAddressGroupMemberHttpUrl
+	getIPAddressGroupMemberPath = strings.ReplaceAll(getIPAddressGroupMemberPath, "{project_id}", getIPAddressGroupMemberClient.ProjectID)
+
+	getIPAddressGroupMemberqueryParams := buildGetIPAddressGroupMemberQueryParams(state)
+	getIPAddressGroupMemberPath += getIPAddressGroupMemberqueryParams
+
+	getIPAddressGroupMemberOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		OkCodes: []int{
+			200,
+		},
+	}
+	getIPAddressGroupMemberResp, err := getIPAddressGroupMemberClient.Request("GET", getIPAddressGroupMemberPath, &getIPAddressGroupMemberOpt)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving IPAddressGroupMember: %s", err)
+	}
+	return utils.FlattenResponse(getIPAddressGroupMemberResp)
+}
+
+func TestAccIPAddressGroupMember_basic(t *testing.T) {
+	var obj interface{}
+
+	name := acceptance.RandomAccResourceName()
+	rName := "huaweicloud_cfw_ip_address_group_member.test"
+
+	rc := acceptance.InitResourceCheck(
+		rName,
+		&obj,
+		getIPAddressGroupMemberResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testIPAddressGroupMember_basic(name),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttrPair(rName, "group_id", "huaweicloud_cfw_ip_address_group.test", "id"),
+					resource.TestCheckResourceAttr(rName, "name", name),
+					resource.TestCheckResourceAttr(rName, "address", "192.168.0.1"),
+					resource.TestCheckResourceAttr(rName, "address_type", "0"),
+				),
+			},
+			{
+				ResourceName:      rName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: testIPAddressGroupMemberImportState(rName),
+			},
+		},
+	})
+}
+
+func testIPAddressGroupMember_basic(name string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "huaweicloud_cfw_ip_address_group_member" "test" {
+  group_id = huaweicloud_cfw_ip_address_group.test.id
+  name = "%s"
+  address = "192.168.0.1"
```

Comment:
modify the format to ensure equal sign alignment

Reply:
fixed